src/app/signalen/contact/page.tsx

```diff
@@ -17,28 +17,22 @@ function Contact() {
       <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
         <form className="ams-gap--md" onSubmit={handleSubmit}>
           <Heading>Doe een melding</Heading>
-
           <Heading level={2}>Contactgegevens</Heading>
-
           <Paragraph size="large">Mogen we u bellen voor vragen? En op de hoogte houden via e-mail?</Paragraph>
-
           <Paragraph>
             Vaak hebben we nog een vraag. Daarmee kunnen we het probleem sneller of beter oplossen. Of we willen iets
             uitleggen. Wij willen u dan graag even bellen. Of anders e-mailen wij u. Wij gebruiken uw telefoonnummer en
             e-mailadres alléén voor deze melding.
           </Paragraph>
-
-          <div>
+          <Field>
             <Label htmlFor="phone">Wat is uw telefoonnummer?(niet verplicht)</Label>
             <input type="tel" id="phone" />
-          </div>
-
-          <div>
+          </Field>
+          <Field>
             <Label htmlFor="mail">Wat is uw e-mailadres?(niet verplicht)</Label>
             <input type="email" id="mail" />
-          </div>
-
-          <Fieldset legend="Mogen we uw melding doorsturen?">
+          </Field>
+          <FieldSet legend="Mogen we uw melding doorsturen?">
             <Paragraph className="ams-mb--md">
```

Comment:
This Paragraph should be connected to the FieldSet as well

Reply:
Yes, added.